packages/dashboard/src/dashboard-actions/reverseAction.ts

```diff
@@ -0,0 +1,67 @@
+import { DashboardConfiguration, MoveAction, ResizeAction } from '../types';
+import { getMovedDashboardConfiguration } from './move';
+import { DashboardAction } from './actions';
+import { resize } from './resize';
+
+export const reverseAction = (dashAction: DashboardAction): DashboardAction | MoveAction | ResizeAction => {
+  switch (dashAction.type) {
+    case 'MOVE': {
+      //create new move action to return later
+      const newMoveAction: MoveAction = dashAction;
+      if (typeof dashAction.payload.prevPosition != 'undefined') {
+        //flip previous and current positions
+        const tempPosition = dashAction.payload.position;
+        newMoveAction.payload.position = dashAction.payload.prevPosition;
+        newMoveAction.payload.prevPosition = tempPosition;
+        return newMoveAction;
+      }
+
+      return newMoveAction;
+    }
+    case 'RESIZE': {
+      const newResizeAction: ResizeAction = dashAction;
+      //invert changeInPosition values
+      newResizeAction.payload.changeInPosition.x = dashAction.payload.changeInPosition.x * -1;
+      newResizeAction.payload.changeInPosition.y = dashAction.payload.changeInPosition.y * -1;
+      return newResizeAction;
+    }
+    default:
+      return dashAction;
+  }
+};
+
+export const applyReverseAction = (
+  dashAction: DashboardAction | MoveAction | ResizeAction,
```

Comment:
`dash` is not a good shorthand for dashboard. Just call it `dashboardAction`

Also why isn't `MoveAction` and `ResizeAction` part of `DashboardAction`?